nginx.conf

```diff
@@ -0,0 +1,35 @@
+error_log /dev/stderr;
+daemon off;
+pid /tmp/nginx_pid;
+
+events {}
+
+http {
+    access_log /dev/stderr;
+    server {
+        listen 8000;
+
+        client_body_temp_path /tmp/nginx_client_temp;
+        fastcgi_temp_path /tmp/nginx_fastcgi_temp;
+        proxy_temp_path /tmp/nginx_proxy_temp;
+        scgi_temp_path /tmp/nginx_scgi_temp;
+        uwsgi_temp_path /tmp/nginx_uwsgi_temp;
```

Comment:
probably safe to delete these things right?

Reply:
Nope, this is standard in all of our static sites and is necessary. nginx errors out without them.